config/routes.rb

```diff
@@ -48,6 +48,8 @@
 
   resources :quick_tips, path: 'caste-otazky', only: :show
 
+  get "rozpracovane-zivotne-situacie" => "user_journeys#in_process", as: "in_process"
```

Comment:
Toto presunme vyssie kedze to patri pod resource `:user_journeys`  dole a mozeme to rovno spravit "restful". A rovno to bude pod /moje-zivotne-situacie/rozpracovane. mozeme to dat rovno pod `index` akciu.